.github/workflows/rubocop.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu ]
-        ruby: [ 2.4, 2.7 ]
+        ruby: [ 2.4, 3.0 ]
         rubocop: [ 'v0.92.0', master ]
```

Comment:
Ruby 3.0 & v0.92 are an issue because of `SafeYAML`. This was resolved in v1.3.0, so maybe we could test v0.92.0 in Ruby 2.4, and 1.3.0 in Ruby 2.4 and 3.0?

Reply:
Why do we even care about 0.92 at this point?

Reply:
True, we don't really need to, we could also simply bump the minimal rubocop version, and we have a good mechanism to do so now.

Reply:
The version specification in YAML should be string `'3.0'` instead of float `3.0`. It will be `3` instead of the expected `3.0`.

<img width="844" alt="result-of-ci" src="https://user-images.githubusercontent.com/13203/103376077-041cd580-4b1f-11eb-8171-ea94b8d937c8.png">